src/index.html

```diff
@@ -16,17 +17,36 @@
       href="style.css"
     />
   </head>
+  <!-- #endregion -->
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        name="big-search"
+        placeholder="Try “Los Angeles”"
+        required=""
+        class="search-bar search-bar__big"
+        data-qa="keypress"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="post"
+      data-qa="small"
+    >
+      <input
+        type="text"
+        name="big-search"
+        placeholder="Try “Los Angeles”"
+        required=""
```

Comment:
The `required` attribute should not be an empty string. In HTML5, it is a boolean attribute and should be written as `required` without any value.